Lib/idlelib/autocomplete.py

```diff
@@ -171,10 +172,13 @@ def fetch_completions(self, what, mode):
                                      (what, mode), {})
         else:
             if mode == ATTRS:
-                if what == "":
+                if what == "":  # Main module names.
                     namespace = {**__main__.__builtins__.__dict__,
                                  **__main__.__dict__}
                     bigl = eval("dir()", namespace)
+                    kwds = (s for s in keyword.kwlist
+                            if s not in {'True', 'False', 'None'})
+                    bigl.extend(kwds)
```

Comment:
As you said on [issue 27609](https://bugs.python.org/issue27609), this method needs to be refactored, so the looks good for adding the keywords and anything else can be handled on the refactor.

Reply:
The refactoring of fetch_completions is issue 37766.  I attached a refactoring of the computation of bigl.  I did not use it here because the computation of smalll for imported modules containing __all__ uses 'namespace' and changing the smalll code is out of scope here.  (Otherwise, I would have.)  Keywords cannot complete module.?.